Library/Formula/pilot-link.rb

```diff
@@ -0,0 +1,22 @@
+class PilotLink < Formula
+  homepage "http://www.pilot-link.org/"
+  url "http://downloads.pilot-link.org/pilot-link-0.12.5.tar.gz"
+  version "0.12.5"
+  sha256 "159b84cadf5cdb39e238feaa652e98fc289a5624356a8ddd7f84d6a55eb5c736"
+
+  depends_on "popt" => :build
+  depends_on "libpng12" => :build
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-conduits",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/pilot-xfer --version"
```

Comment:
Can this test be made more complex? If not, style should be: ` system "#{bin}/pilot-xfer", "--version" `.